### PR TITLE
Added {{one-way-select}}

### PR DESCRIPTION
**Pull request by martndemus**
In advance of the `one-way-controls` rename.

### CODE REVIEW

File: addon/components/one-way-select.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    'multiple',
+    'name',
+    'required',
+    'size'
+  ],
+
+  didReceiveAttrs() {
+    this._super(...arguments);
+
+    let options = this.getAttr('options');
```
Comment (poteto) [inline addon/components/one-way-select.js]: Can we use a regular `get` here?

File: addon/components/one-way-select.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    } else {
+      value = this._selectedSingle();
+    }
+
+    invokeAction(this, 'update', value);
+  },
+
+  prompt: alias('includeBlank'),
+
+  promptText: computed('prompt', function() {
```
Comment (poteto) [inline addon/components/one-way-select.js]: `prompt` isn't being consumed by this CP

File: package.json [hunk 3]
```diff
@@ -37,6 +37,8 @@
     "ember-export-application-global": "^1.0.4",
     "ember-disable-prototype-extensions": "^1.0.0",
     "ember-try": "~0.0.8"
+    "ember-invoke-action": "1.2.0",
```
Comment (poteto) [inline package.json]: Missing comma on L39

File: addon/components/one-way-select.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+const {
+  computed,
+  computed: { alias },
+  get,
+  set,
+  String: { w }
+} = Ember;
+
+export default Ember.Component.extend({
```
Comment (poteto) [inline addon/components/one-way-select.js]: We can destructure `Component`

File: addon/components/one-way-select.js [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    let options = this.getAttr('options');
+    if (typeof options === 'string') {
+      options = w(options);
+    }
+
+    set(this, 'options', Ember.A(options));
+  },
+
+  optionGroups: computed('options.[]', function() {
+    const groupLabelPath = get(this, 'groupLabelPath');
```
Comment (poteto) [inline addon/components/one-way-select.js]: Prefer not to use this kind of alignment, makes it annoying to add new variables

File: tests/integration/components/one-way-select-test.js [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+    this.set('value', 'female');
+    this.set('options', ['unknown', 'male', 'female']);
+  }
+});
+
+test('It renders a select box with options', function(assert) {
+  this.render(hbs`
+    {{one-way-select value=value options=options}}
+  `);
+  assert.equal(this.$('option').length, 3);
```
Comment (poteto) [inline tests/integration/components/one-way-select-test.js]: Can we add assertion messages to all our asserts in this test?

**Comment by martndemus** [comment]
@poteto Updated per feedback.

**Comment by martndemus** [comment]
@poteto What are we going to do about 1.13.x failing? Should we recommend people install the get helper add-on?

**Comment by poteto** [comment]
We can include it as an npm dependency to polyfill it.

**Comment by martndemus** [comment]
Works for me... Added!

**Comment by poteto** [comment]
LGTM, can you also add an export to [`index.js`](https://github.com/dockyard/ember-one-way-controls/blob/master/addon%2Findex.js) so it's easier to import in consuming apps?

**Comment by poteto** [comment]
@homu r+

**Pull request merged**